rm-deployment-status-enhancer-summary/bower.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Can you come up with a shorter name for this folder? "rm-deployment-status-enhancer-summary" is really long. One option is to have a "release-management" root folder, that may contain one or more folders with different samples. It a more sustainable model ... and one we will start moving towards as we get more samples.